### PR TITLE
Use current major Shopware version 5.6

### PR DESCRIPTION
**Pull request by larsbo**

### CODE REVIEW

**Comment by jkrzefski** [comment]
Please also include https://github.com/larsbo/composer-project/pull/1 into this pull request.

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": "^7.0",
```
Comment (jkrzefski) [inline composer.json]: Maybe also change this to `^7.2` as it is in `shopware/shopware`. What do you think?

Comment (shyim) [reply composer.json]: Yep!

**Comment by shyim** [comment]
Thanks for your Contribution! :blue_heart:

**Pull request merged**